scripts/verify_semver.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
```

Comment:
do not validate semver, if anything then maybe verify if that image with this tag were already pulbished